package.json

```diff
@@ -29,8 +29,8 @@
     "prepare": "rm -rf lib && tsc -p . && tsc -p tsconfig.es5.json",
     "prettier:check": "prettier --check 'src/{main,tests}/**/*.{json,ts,tsx,js,jsx,css,scss,html}'",
     "prettier:write": "prettier --write 'src/{main,tests}/**/*.{json,ts,tsx,js,jsx,css,scss,html}'",
-    "test": "jest --maxWorkers=50%",
-    "test:coverage": "jest --collectCoverage --maxWorkers=50%",
+    "test": "jest -w=4",
+    "test:coverage": "jest --collectCoverage -w=4",
```

Comment:
What does this do? Is there some longer flag name so that it's obvious more quickly?

Reply:
--maxWorkers=50% = uses 50% of available cores
-w = uses 4 cores

Reply:
I think you can do `--maxWorkers=4`, which would be more readable and [is just an alias](https://jestjs.io/docs/cli#--maxworkersnumstring) 🤔